### PR TITLE
Update Readme for Sql Export Module

### PR DESCRIPTION
**Pull request by bealdav**
Just more explanation

### CODE REVIEW

**Comment by bealdav** [comment]
ping @benoitguillot @hparfr @aheficent

File: sql_export/README.rst [hunk 1]
```diff
@@ -37,6 +37,16 @@ Dashboards > Sql Export.
...
     :target: https://runbot.odoo-community.org/runbot/149/10.0
+    
+
+Specific use with parameters
+----------------------------
+
+- `%(company_id)s` allow to set in the query the company id of the user
+- `%(user_id)s` allow to set in the query the user id
+- for any created field with `Sql Export Variables` menu, you can use it with `%(x_field_example)s` syntax 
+  (Limitiation for relationnal fields)
```
Comment (hparfr) [inline sql_export/README.rst]: %s/limitiation/limitation
%s/relationnal/relational

**Review by hparfr — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Thanks

File: sql_export/README.rst [hunk 2]
```diff
@@ -37,6 +37,16 @@ Dashboards > Sql Export.
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
     :alt: Try me on Runbot
     :target: https://runbot.odoo-community.org/runbot/149/10.0
+    
+
+Specific use with parameters
+----------------------------
+
+- `%(company_id)s` allow to set in the query the company id of the user
```
Comment (bguillot) [inline sql_export/README.rst]: allows instead of allow

File: sql_export/README.rst [hunk 3]
```diff
@@ -37,6 +37,16 @@ Dashboards > Sql Export.
...
+- `%(user_id)s` allow to set in the query the user id
```
Comment (bguillot) [inline sql_export/README.rst]: allows instead of allow

**Review by bguillot — APPROVED** [review]
Thanks !

**Comment by bealdav** [comment]
ping @pedrobaeza could you merge this trivial PR. Thanks

**Pull request merged**